deps/chakrashim/src/inspector/v8-inspector-session-impl.cc

```diff
@@ -28,7 +31,9 @@ bool V8InspectorSession::canDispatchMethod(const StringView& method) {
          stringViewStartsWith(method,
                               protocol::Console::Metainfo::commandPrefix) ||
          stringViewStartsWith(method,
-                              protocol::Schema::Metainfo::commandPrefix);
+                              protocol::Schema::Metainfo::commandPrefix) ||
+         stringViewStartsWith(method,
```

Comment:
>stringViewStartsWith [](start = 9, length = 20)

not too familiar with this code, but does this have to be inside TTT if-condition?